src/index.html

```diff
@@ -3,12 +3,38 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@500&display=swap">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo-link" href="#"> <img class="logo" src="images/logo.png"
+         alt="LOGO">
+      </a>
```

Comment:
```suggestion
      <a class="logo-link" href="#"> 
        <img class="logo" 
          src="images/logo.png"
          alt="LOGO"
        >
      </a>
```

You try to have 2 different tags in 1 line